app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -36,23 +36,23 @@ class PlanWizardInstancePropertiesStepTable extends React.Component {
 
   // enables our custom header formatters extensions to reactabular
   customHeaderFormatters = Table.customHeaderFormattersDefinition;
-
-  // TODO:  These methods need to be wired up to redux-form once we know what
-  //        the POST body should look like for creating an OSP transformation
-  //        plan.  We will use local state to store the row that is currently
-  //        being edited to roll back to in case of cancel.
   inlineEditController = () => {
-    const { rows } = this.props;
+    const { rows, updatedInstancePropertiesRowOnStandby, instancePropertiesRowsAction } = this.props;
     return {
       isEditing: ({ rowData }) => rowData.id === this.state.backup.id,
       onActivate: ({ rowData }) => {
         const index = findIndex(rows, { id: rowData.id });
         const backup = rows[index];
 
         this.setState({ backup, editing: true });
+        const { setUpdatedRowOnStandbyAction } = this.props;
+        setUpdatedRowOnStandbyAction(rowData);
```

Comment:
I like that we're storing the backup row in redux here, b/c, that'll allow us to remove our dependency on `findIndex` :grin:

Can we please yank that from `package.json`, remove all instances of `findIndex()` from `inlineEditController`, remove `backup` from component state, and add `setUpdatedRowOnStandbyAction({})` to `onConfirm`, and `onCancel`

:tada: